Basic/DynamicAddressablesNetworkPrefabs/Assets/Scripts/02_Server Authoritative Load All Async/ServerAuthoritativeLoadAllAsync.cs

```diff
@@ -161,18 +193,28 @@ void LoadAddressableClientRpc(AddressableGUID guid, ClientRpcParams rpcParams =
 
             async void Load(AddressableGUID assetGuid)
             {
+                // loading prefab as a client, update UI
+                m_InGameUI.ClientLoadedPrefabStatusChanged(m_NetworkManager.LocalClientId, assetGuid.GetHashCode(), "Undefined", InGameUI.LoadStatus.Loading);
+                
                 Debug.Log("Loading dynamic prefab on the client...");
-                await DynamicPrefabLoadingUtilities.LoadDynamicPrefab(assetGuid, m_ArtificialDelayMilliseconds);
+                await DynamicPrefabLoadingUtilities.LoadDynamicPrefab(assetGuid, m_InGameUI.ArtificialDelayMilliseconds);
                 Debug.Log("Client loaded dynamic prefab");
-                AcknowledgeSuccessfulPrefabLoadServerRpc(assetGuid.GetHashCode());
+                
+                DynamicPrefabLoadingUtilities.TryGetLoadedGameObjectFromGuid(assetGuid, out var loadedGameObject);
+                m_InGameUI.ClientLoadedPrefabStatusChanged(m_NetworkManager.LocalClientId, assetGuid.GetHashCode(), loadedGameObject.Result.name, InGameUI.LoadStatus.Loaded);
+                
+                AcknowledgeSuccessfulPrefabLoadServerRpc(assetGuid.GetHashCode(), loadedGameObject.Result.name);
             }
         }
         
         [ServerRpc(RequireOwnership = false)]
-        void AcknowledgeSuccessfulPrefabLoadServerRpc(int prefabHash, ServerRpcParams rpcParams = default)
+        void AcknowledgeSuccessfulPrefabLoadServerRpc(int prefabHash, string loadedPrefabName, ServerRpcParams rpcParams = default)
```

Comment:
I wouldn't use bandwidth with prefab name strings just for debug specific ui updates. This would corrupt the main flows we're trying to show.
I'm ok with adding UI notif noise in other parts of the code, but this is core education, this shouldn't be touched